test/helpers/config_scanning.rb

```diff
@@ -30,6 +35,9 @@ def scan_and_remove_used_entries(default_keys, non_test_files)
             captures.flatten.compact.each do |key|
               default_keys.delete(key.delete("'").to_sym)
             end
+
+            IGNORED.each { |key| default_keys.delete(key) }
```

Comment:
This situation is something this test already accounts for by checking for `dynamic name` on the config a few lines below this, as described by the comment below. Instead of adding a second way to accomplish this, we should update the config in default source to include `dynamic_name => true`.

Reply:
Whoops! We need a TTRM for "Today Tanna Reminded Me" (similar to TIL for Today I Learned)! Addressed with [fa500e4](https://github.com/newrelic/newrelic-ruby-agent/pull/2177/commits/fa500e436fced67a32e7fea7d06dcbd739a97b0e)